src/main.py

```diff
@@ -1,3 +1,9 @@
-from face_detect import detect_function
-
-detect_function() #calling function to recognize the face in the image
\ No newline at end of file
+from face_detect import image_detect,video_detect,cam_detect
+import sys
+    
+if sys.argv[0] == 1:
```

Comment:
Try using argparse library for more verbose arguments to run the program. All Unix commands demonstrate what good command line arguments look like.